### PR TITLE
Parameterize and add KMS rule file

### PR DESCRIPTION
**Pull request by hshin-g**
- Parameterize all user configurable values in forseti_conf_server.yaml.tpl
- Add in KMS scanner rule file into rules directory

### CODE REVIEW

File: modules/server/templates/configs/forseti_conf_server.yaml.tpl [hunk 1]
```diff
@@ -374,8 +376,28 @@ notifier:
                 # gcs_path should begin with "gs://"
                 gcs_path: gs://${FORSETI_BUCKET}/scanner_violations
 
-        - resource: groups_violations
+        - resource: kms_violations
           should_notify: true
+          notifiers: ${KMS_VIOLATIONS_SHOULD_NOTIFY}
+            # Email violations
```
Comment (adrienthebo) [inline modules/server/templates/configs/forseti_conf_server.yaml.tpl]: It looks like there's a two space indentation here which will cause this to be invalid YAML

Comment (hshin-g) [reply modules/server/templates/configs/forseti_conf_server.yaml.tpl]: Resolved- thanks Adrien.

File: variables.tf [hunk 2]
```diff
@@ -79,6 +79,458 @@ variable "server_instance_metadata" {
...
+
+#-------------------------------#
+# Forseti server config scanner #
+#-------------------------------#
+variable "audit_logging_enabled" {
+  description = "Audit Logging scanner enabled."
+  default     = "false"
+}
+
+variable "biquery_enabled" {
```
Comment (adrienthebo) [inline variables.tf]: Typo - this should be `bigquery_enabled`, we dropped the first g in `bigquery`

Comment (hshin-g) [reply variables.tf]: Good eye. Changed.

**Pull request merged**